wit/surface.wit

```diff
@@ -1,23 +1,24 @@
 package wasi:webgpu;
 
 interface surface {
-    use graphics-context.{graphics-context};
+    use graphics-context.{context};
     use wasi:io/poll@0.2.0.{pollable};
 
     record create-desc {
-        height: u32,
-        width: u32,
-        offscreen: bool,
+        height: option<u32>,
```

Comment:
Atanas has mentioned the necessity of "windowless" processes for AI workloads - would empty height and width signify this at creation time? Or is that outside of the scope of this API?

Reply:
For windowsless we don't need the graphics-context and surface at all. Can be done just with webgpu.

Reply:
Makes sense - you can initialize webgpu directly and not connect it, and it will work headless